docs/development.md

```diff
@@ -41,4 +41,28 @@ grammar](https://code.visualstudio.com/api/language-extensions/syntax-highlight-
 trigger the [scope
 inspector](https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#scope-inspector)
 from the Command Palette (`Ctrl+Shift+P`) with the `Developer: Inspect Editor
-Tokens and Scopes` command.
\ No newline at end of file
+Tokens and Scopes` command.
+
+## Publishing the extension in VS Code Marketplace
+
+The publisher ID we are using is "UCLResearchSoftwareDevelopmentGroup", and it's linked to RSDG's notifications email account. You can see login details for it in RSDG's shared passwords list in LastPass.
+
+Then, update Nisaba's version number in `package.json` for the new release.
+
+Once that's done, you can locally run:
+
+```
+npm run publish
+```
+
+which will package and publish the extension in the Marketplace.
+
+Please note you'll be prompted to enter a Personal Account Token (PAT) when publishing the extension. The one we are using is currently saved in LastPass. It will expire in one year (27th October 2022). Instructions to create a new one are also in the LastPass entry.
+
+If you make a mistake and want to update your release maintaining the same version number, you'll have to first unpublish the release you just made:
+
+```
+npm run unpublish
+```
+
+After that you can publish again as normal.
```

Comment:
With `unpublish` you unpublish a single version or the entire extension with all previous versions?  In the [Marketplace](https://marketplace.visualstudio.com/items?itemName=UCLResearchSoftwareDevelopmentGroup.nisaba) I can't see v0.0.1 anymore, it's fine for that version, but I guess that going forward we don't want to wipe everything out every time.

Reply:
Looks like you unpublish the whole extension. In fact in their guidelines it says:

> Note: When you unpublish an extension, the Marketplace will remove any extension statistics it has collected. You may want to update your extension rather than unpublish it.

It does seem to be a command that suits this "update" process they mention, so in this case we'd still need to log in and do everything online. I'll amend the docs accordingly.